grails-app/views/tree/_adminTableRow.gsp

```diff
@@ -0,0 +1,124 @@
+<tr id="summary-${tree.id}">
+    <td width="40%">
+        <div>
+            <div class="btn btn-link" onclick="showInfo('${tree.id}')">
+                <g:if test="${tree.expertTree}">
+                    <span class="fa fa-mortar-board" title="Expert recommended tree">&nbsp;&nbsp;</span>
+                </g:if>
+                <g:else>
+                    <span class="fa fa-users" title="Public tree">&nbsp;&nbsp;</span>
+                </g:else>
+                ${tree.getTitle()}
+                <i class="icon-info-sign" title="Show more information"></i>
+            </div>
+        </div>
+    </td>
+    <td width="30%"><a href="http://bie.ala.org.au/species/${tree.expertTreeLSID}"
+                       target="_blank">${tree.getExpertTreeTaxonomy()}</a></td>
+    <td width="30%">
+        <a id="showHideDetailsBtn${tree.id}" class="btn btn-small btn-primary" data-toggle="modal" href="#"
+           onclick="showInfo('${tree.id}')">
+            <i class="fa fa-angle-double-down"></i>&nbsp;&nbsp;Details</a>
+
+        <a href="${createLink(controller: 'viewer', action: 'show')}?studyId=${tree.getId()}"
+           class="btn btn-small"><i class="icon icon-camera"></i> Preview tree</a>
+    </td>
+</tr>
+<tr id="info-${tree.id}" class="hide">
+    <td colspan="3" class="info" width="100%">
+        <i>${tree.getReference()}</i>
+        <g:if test="${tree.doi}">
+            <div class="">
+                <a href="${tree.getDoi()}">${tree.getDoi()}</a>
+            </div>
+        </g:if>
+    </td>
+</tr>
+<tr id="controls-${tree.id}" class="hide" width="100%">
+    <td colspan="3" class="info">
+        <g:if test="${tree.expertTree}">
+            <g:remoteLink params="[treeId: tree.id]" class="btn btn-primary btn-small"
+                          action="toggleExpertTree" update="row${tree.id}"
+                          title="Remove this tree from the list of expert trees' : ''}">
+                Change to Public Tree
+            </g:remoteLink>
+        </g:if>
+        <g:else>
+            <g:formRemote name="bla" url="[action: 'toggleExpertTree']" controller="tree" action="toggleExpertTree"
+                          method="POST" update="row${tree.id}"
+                          onComplete="jQuery('#promoteExpertTreeModal${tree.id}').modal('hide')">
+                <p>Promote this tree as an 'expert recommended tree'.</p>
+
+                <div class="form-horizontal">
+                    <div class="control-group">
+                        <label for="expertTreeTaxonomy" class="control-label">Species covered</label>
```

Comment:
we can reduce this to just the autocomplete box. I do not think we will have a situation where the name is different from linked profile name.